docs/source/developer-guidelines/images.md

```diff
@@ -0,0 +1,65 @@
+# Images
+
+Images are rendered with the Volto component `Image`.  
+It renders images using Plone scales from [plone.app.imaging](https://github.com/plone/plone.app.imaging) and handles:
```

Comment:
Minor documentation comment: `plone.app.imaging` was only used for Archetypes, so not relevant here. Scales for dexterity are done in `plone.namedfile`, so the link should point there.